[READY] Cargo packet update and modification

Trilbyspaceclone opened the pull request:
[Changelogs
Adds in 4 more crates
Medical Misc Kit, one of each medical kits - 1250
Medical sprays, two of each medical sprays - 2500
WT-550 Auto Rifle ammo crates non-lethal - 1500
WT-550 Auto Rifle ammo  Special -3000
Adds stimulant pills to medical gear crate
Removes lots of different things in crates
Adds in tons of different things in crates
[why]
Makes makes crates that were over priced more reachable , and adds in more items that should be in the crates.

Trilbyspaceclone committed:
Adds memes

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -393,7 +396,10 @@
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt)
```
There's already a helmet crate

Trilbyspaceclone replied:
https://cdn.discordapp.com/attachments/499347841756561436/501029245057892354/unknown.png
No there isnt.

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -419,9 +425,10 @@
-	desc = "Contains two Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
-	cost = 2500
+	desc = "Contains three Energy Guns, capable of firing both nonlethal and lethal blasts of light. Requires Armory access to open."
+	cost = 2750
```
That's a pretty minor price increase for a 1.5x increase in total firepower

Trilbyspaceclone replied:
Thats now how 1.5x works

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -465,7 +472,9 @@
-					/obj/item/grenade/chem_grenade/incendiary)
+					/obj/item/grenade/chem_grenade/incendiary,
+					/obj/item/clothing/suit/fire/atmos,
+					/obj/item/clothing/head/hardhat/atmos)
```
Why tho. There's already a firesuit crate

Trilbyspaceclone replied:
Thats for sec, so they can use it without dieing form fire...

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -534,13 +543,32 @@
+/datum/supply_pack/security/armory/wt550ammo_special
+	name = "WT-550 Auto Rifle Special Ammo Crate"
+	desc = "Contains 2 20-round Armour Piercing and Incendiary magazines for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
+	cost = 3000
```
Pretty cheap for some pretty deadly rounds

Trilbyspaceclone replied:
KK ill up the cost to 4500

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -1324,7 +1376,8 @@
 					/obj/item/grenade/chem_grenade/cleaner,
-					/obj/item/grenade/chem_grenade/cleaner)
+					/obj/item/grenade/chem_grenade/cleaner,
+					/obj/item/soap/deluxe)
```
Why tho. The entire point of the deluxe soap is that it's captain-exclusive

Trilbyspaceclone replied:
There are many ways to get the soap, meta station has some in maints, you can get it form space, and some maps have them in bathrooms I think

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -1544,8 +1604,7 @@
 					/obj/item/pizzabox/mushroom,
 					/obj/item/pizzabox/meat,
-					/obj/item/pizzabox/vegetable,
-					/obj/item/pizzabox/pineapple)
```
🗡

Trilbyspaceclone replied:
<3

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -2135,7 +2225,6 @@
 					/obj/item/pen/fourcolor,
 					/obj/item/pen/fourcolor,
 					/obj/item/pen,
-					/obj/item/pen/fountain,
```
Why tho

Trilbyspaceclone replied:
Fountain pens are for the Calligraphy Crate

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -2144,14 +2233,16 @@
 	cost = 700
-	contains = list(/obj/item/storage/box/fountainpens)
+	contains = list(/obj/item/storage/box/fountainpens,
+					/obj/item/paper_bin)
```
why tho

Trilbyspaceclone replied:
500 for 7 pens is kinda useless

deathride58 reviewed (changes requested):
See above

Trilbyspaceclone committed:
Fixed

Trilbyspaceclone committed:
But why tho?

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -129,9 +129,10 @@
 	name = "Metal Foam Grenade Crate"
-	desc = "Seal up those pesky hull breaches with 7 Metal Foam Grenades."
+	desc = "Seal up those pesky hull breaches with 14 Metal Foam Grenades."
 	cost = 1000
```
Could at least double the cost of the crate to account for the doubled count of metal foam nades
```suggestion
	cost = 2000
```

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -393,7 +396,10 @@
-					/obj/item/clothing/suit/armor/bulletproof)
+					/obj/item/clothing/suit/armor/bulletproof,
+					/obj/item/clothing/head/helmet/alt,
+					/obj/item/clothing/head/helmet/alt,
```
Make a separate bulletproof helmet crate or double the cost

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -450,13 +457,13 @@
 /datum/supply_pack/security/armory/fire
 	name = "Incendiary Weapons Crate"
-	desc = "Burn, baby burn. Contains three incendiary grenades, three plasma canisters, and a flamethrower. Requires Armory access to open."
+	desc = "Burn, baby burn. Contains three incendiary grenades, three plasma canisters, an atmos fire suit and a flamethrower. Requires Armory access to open."
```
Description needs adjustment due to the removal of the atmos fire suit from the crate

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -1303,14 +1352,15 @@
 /datum/supply_pack/service/janitor
 	name = "Janitorial Supplies Crate"
-	desc = "Fight back against dirt and grime with Nanotrasen's Janitorial Essentials(tm)! Contains three buckets, caution signs, and cleaner grenades. Also has a single mop, spray cleaner, rag, and trash bag."
+	desc = "Fight back against dirt and grime with Nanotrasen's Janitorial Essentials(tm)! Contains three buckets, caution signs, and cleaner grenades. Also has a single mop, spray cleaner, rag, deluxe soap and a trash bag."
```
Description needs adjustment

Trilbyspaceclone committed:
Fixes what Bejihn wanted and followed the sitcky at the top

"// If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal"

Trilbyspaceclone committed:
Removes a spare thingy

Trilbyspaceclone committed:
Sigh

deathride58 commented on code/modules/cargo/packs.dm:
```diff
@@ -396,6 +399,15 @@
 					/obj/item/clothing/suit/armor/bulletproof)
 	crate_name = "bulletproof armor crate"
 
+/datum/supply_pack/security/armory/bulletarmor
```
Duplicate typepath.
```suggestion
/datum/supply_pack/security/armory/bullethelmets
```

deathride58 committed:
Okie

Co-Authored-By: Trilbyspaceclone <30435998+Trilbyspaceclone@users.noreply.github.com>
